2016-01-14T20:19:33.171674+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/site-packages/django/__init__.py", line 18, in setup
2016-01-14T20:19:33.171681+00:00 app[web.1]: Traceback (most recent call last):
2016-01-14T20:19:33.171678+00:00 app[web.1]:     module = import_module(entry)
2016-01-14T20:19:33.171679+00:00 app[web.1]:     return _bootstrap._gcd_import(name[level:], package, level)
2016-01-14T20:19:33.171678+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/importlib/__init__.py", line 109, in import_module
2016-01-14T20:19:33.171681+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/site-packages/gunicorn/arbiter.py", line 515, in spawn_worker
2016-01-14T20:19:33.171675+00:00 app[web.1]:     apps.populate(settings.INSTALLED_APPS)
2016-01-14T20:19:33.171680+00:00 app[web.1]: ImportError: No module named 'bootstrap_toolkit'
2016-01-14T20:19:33.171682+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/site-packages/gunicorn/workers/base.py", line 122, in init_process
2016-01-14T20:19:33.171682+00:00 app[web.1]:     worker.init_process()
2016-01-14T20:19:33.171683+00:00 app[web.1]:     self.load_wsgi()
2016-01-14T20:19:33.171684+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/site-packages/gunicorn/workers/base.py", line 130, in load_wsgi
2016-01-14T20:19:33.171685+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/site-packages/gunicorn/app/base.py", line 67, in wsgi
2016-01-14T20:19:33.171684+00:00 app[web.1]:     self.wsgi = self.app.wsgi()
2016-01-14T20:19:33.171686+00:00 app[web.1]:     self.callable = self.load()
2016-01-14T20:19:33.171694+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/site-packages/django/__init__.py", line 18, in setup
2016-01-14T20:19:33.171688+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
2016-01-14T20:19:33.171691+00:00 app[web.1]:   File "/app/practica4/wsgi.py", line 16, in <module>
2016-01-14T20:19:33.171691+00:00 app[web.1]:     application = get_wsgi_application()
2016-01-14T20:19:33.171693+00:00 app[web.1]:     django.setup()
2016-01-14T20:19:33.171689+00:00 app[web.1]:     return util.import_app(self.app_uri)
2016-01-14T20:19:33.171692+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/site-packages/django/core/wsgi.py", line 13, in get_wsgi_application
2016-01-14T20:19:33.171687+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
2016-01-14T20:19:33.171687+00:00 app[web.1]:     return self.load_wsgiapp()
2016-01-14T20:19:33.171690+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/site-packages/gunicorn/util.py", line 357, in import_app
2016-01-14T20:19:33.171690+00:00 app[web.1]:     __import__(module)
2016-01-14T20:19:33.171694+00:00 app[web.1]:     apps.populate(settings.INSTALLED_APPS)
2016-01-14T20:19:33.171696+00:00 app[web.1]:     app_config = AppConfig.create(entry)
2016-01-14T20:19:33.171697+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/site-packages/django/apps/config.py", line 90, in create
2016-01-14T20:19:33.171698+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/importlib/__init__.py", line 109, in import_module
2016-01-14T20:19:33.171697+00:00 app[web.1]:     module = import_module(entry)
2016-01-14T20:19:33.171699+00:00 app[web.1]:     return _bootstrap._gcd_import(name[level:], package, level)
2016-01-14T20:19:33.171695+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/site-packages/django/apps/registry.py", line 85, in populate
2016-01-14T20:19:33.183835+00:00 app[web.1]: ImportError: No module named 'bootstrap_toolkit'
2016-01-14T20:19:33.193618+00:00 app[web.1]: [2016-01-14 20:19:33 +0000] [7] [ERROR] Exception in worker process:
2016-01-14T20:19:33.193622+00:00 app[web.1]: Traceback (most recent call last):
2016-01-14T20:19:33.193624+00:00 app[web.1]:     worker.init_process()
2016-01-14T20:19:33.193623+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/site-packages/gunicorn/arbiter.py", line 515, in spawn_worker
2016-01-14T20:19:33.193625+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/site-packages/gunicorn/workers/base.py", line 122, in init_process
2016-01-14T20:19:33.193626+00:00 app[web.1]:     self.load_wsgi()
2016-01-14T20:19:33.193626+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/site-packages/gunicorn/workers/base.py", line 130, in load_wsgi
2016-01-14T20:19:33.193627+00:00 app[web.1]:     self.wsgi = self.app.wsgi()
2016-01-14T20:19:33.193628+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/site-packages/gunicorn/app/base.py", line 67, in wsgi
2016-01-14T20:19:33.193629+00:00 app[web.1]:     self.callable = self.load()
2016-01-14T20:19:33.193630+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
2016-01-14T20:19:33.193630+00:00 app[web.1]:     return self.load_wsgiapp()
2016-01-14T20:19:33.193631+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
2016-01-14T20:19:33.193632+00:00 app[web.1]:     return util.import_app(self.app_uri)
2016-01-14T20:19:33.193633+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/site-packages/gunicorn/util.py", line 357, in import_app
2016-01-14T20:19:33.193634+00:00 app[web.1]:     __import__(module)
2016-01-14T20:19:33.193634+00:00 app[web.1]:   File "/app/practica4/wsgi.py", line 16, in <module>
2016-01-14T20:19:33.193635+00:00 app[web.1]:     application = get_wsgi_application()
2016-01-14T20:19:33.193636+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/site-packages/django/core/wsgi.py", line 13, in get_wsgi_application
2016-01-14T20:19:33.193636+00:00 app[web.1]:     django.setup()
2016-01-14T20:19:33.193637+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/site-packages/django/__init__.py", line 18, in setup
2016-01-14T20:19:33.193638+00:00 app[web.1]:     apps.populate(settings.INSTALLED_APPS)
2016-01-14T20:19:33.193638+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/site-packages/django/apps/registry.py", line 85, in populate
2016-01-14T20:19:33.193645+00:00 app[web.1]:     app_config = AppConfig.create(entry)
2016-01-14T20:19:33.193645+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/site-packages/django/apps/config.py", line 90, in create
2016-01-14T20:19:33.193646+00:00 app[web.1]:     module = import_module(entry)
2016-01-14T20:19:33.193646+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/importlib/__init__.py", line 109, in import_module
2016-01-14T20:19:33.193647+00:00 app[web.1]:     return _bootstrap._gcd_import(name[level:], package, level)
2016-01-14T20:19:33.193648+00:00 app[web.1]: ImportError: No module named 'bootstrap_toolkit'
2016-01-14T20:19:33.193648+00:00 app[web.1]: Traceback (most recent call last):
2016-01-14T20:19:33.193649+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/site-packages/gunicorn/arbiter.py", line 515, in spawn_worker
2016-01-14T20:19:33.193650+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/site-packages/gunicorn/workers/base.py", line 122, in init_process
2016-01-14T20:19:33.193649+00:00 app[web.1]:     worker.init_process()
2016-01-14T20:19:33.193651+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/site-packages/gunicorn/workers/base.py", line 130, in load_wsgi
2016-01-14T20:19:33.193651+00:00 app[web.1]:     self.load_wsgi()
2016-01-14T20:19:33.193652+00:00 app[web.1]:     self.wsgi = self.app.wsgi()
2016-01-14T20:19:33.193653+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/site-packages/gunicorn/app/base.py", line 67, in wsgi
2016-01-14T20:19:33.193653+00:00 app[web.1]:     self.callable = self.load()
2016-01-14T20:19:33.193654+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
2016-01-14T20:19:33.193655+00:00 app[web.1]:     return self.load_wsgiapp()
2016-01-14T20:19:33.193668+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
2016-01-14T20:19:33.184340+00:00 app[web.1]: [2016-01-14 20:19:33 +0000] [8] [INFO] Worker exiting (pid: 8)
2016-01-14T20:19:33.199919+00:00 app[web.1]: [2016-01-14 20:19:33 +0000] [7] [INFO] Worker exiting (pid: 7)
2016-01-14T20:19:33.193669+00:00 app[web.1]:     return util.import_app(self.app_uri)
2016-01-14T20:19:33.193670+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/site-packages/gunicorn/util.py", line 357, in import_app
2016-01-14T20:19:33.193670+00:00 app[web.1]:     __import__(module)
2016-01-14T20:19:33.302767+00:00 app[web.1]: [2016-01-14 20:19:33 +0000] [3] [INFO] Reason: Worker failed to boot.
2016-01-14T20:19:33.193671+00:00 app[web.1]:     application = get_wsgi_application()
2016-01-14T20:19:33.193671+00:00 app[web.1]:   File "/app/practica4/wsgi.py", line 16, in <module>
2016-01-14T20:19:33.193672+00:00 app[web.1]:     django.setup()
2016-01-14T20:19:33.193672+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/site-packages/django/core/wsgi.py", line 13, in get_wsgi_application
2016-01-14T20:19:33.193674+00:00 app[web.1]:     apps.populate(settings.INSTALLED_APPS)
2016-01-14T20:19:33.193674+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/site-packages/django/apps/registry.py", line 85, in populate
2016-01-14T20:19:33.193673+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/site-packages/django/__init__.py", line 18, in setup
2016-01-14T20:19:33.193676+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/site-packages/django/apps/config.py", line 90, in create
2016-01-14T20:19:33.193675+00:00 app[web.1]:     app_config = AppConfig.create(entry)
2016-01-14T20:19:33.193676+00:00 app[web.1]:     module = import_module(entry)
2016-01-14T20:19:33.193677+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.4/importlib/__init__.py", line 109, in import_module
2016-01-14T20:19:33.193678+00:00 app[web.1]:     return _bootstrap._gcd_import(name[level:], package, level)
2016-01-14T20:19:33.199441+00:00 app[web.1]: ImportError: No module named 'bootstrap_toolkit'
2016-01-14T20:19:33.298263+00:00 app[web.1]: [2016-01-14 20:19:33 +0000] [3] [INFO] Shutting down: Master
2016-01-14T20:19:34.161914+00:00 heroku[web.1]: State changed from up to crashed
2016-01-14T20:19:34.159820+00:00 heroku[web.1]: Process exited with status 3
2016-01-14T20:19:38.421531+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=desolate-refuge-4895.herokuapp.com request_id=5318abd9-0022-4f1a-89b2-7eaf89a8b9fe fwd="83.53.219.98" dyno= connect= service= status=503 bytes=
2016-01-14T20:19:33.050586+00:00 heroku[web.1]: State changed from starting to up
2016-01-14T20:19:40.628263+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/favicon.ico" host=desolate-refuge-4895.herokuapp.com request_id=af3b800a-78d5-47fd-bd9b-21ffd062015e fwd="83.53.219.98" dyno= connect= service= status=503 bytes=
2016-01-14T20:19:40.770432+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/favicon.ico" host=desolate-refuge-4895.herokuapp.com request_id=30dfcc75-b9ef-47bf-82a8-70f831b39451 fwd="83.53.219.98" dyno= connect= service= status=503 bytes=
